csclient/params/params.go

```diff
@@ -23,6 +23,10 @@ const (
 	// EntityIdHeader specifies the header attribute that will hold the
 	// id of the entity for archive GET responses.
 	EntityIdHeader = "Entity-Id"
+
+	// ResourceRevisionHeader specifies the header attribute that will hold the
+	// revision of the resource retrieved by a resource GET request.
+	ResourceRevisionHeader = "Resource-Revision"
```

Comment:
yeah, I'm just following what we do for returning the entity ID (see above where we get it from a header too).